src/Network/GRPC/MQTT/Compress.hs

```diff
@@ -84,47 +77,21 @@ toRemoteError err =
         , Proto.remoteErrorExtra = Nothing
         }
 
--- | TODO
+-- | Decompress a 'ByteString' (compressed via 'compress') as a single
+-- zstandard frame.
+--
+-- * 'StreamError' is emitted if the 'ByteString' was compressed in streaming
+--   mode.
+--
+-- * 'GenericError' is emitted if an error was thrown by the internal zstandard
+--   FFI call.
```

Comment:
Is this still true with your changes to `ZstdError`?

Reply:
Thanks for catching this, I've updated the documentation for it.